frontend/src/app/blog/[name]/[id]/page.tsx

```diff
@@ -0,0 +1,125 @@
+"use client";
+import Image from "next/image";
+import { useParams } from "next/navigation";
+import BlogsDataJson from "../../../../data/blogs.json";
+import { BlogsDataType } from "@/types";
+
+const Blog = () => {
+  const BlogsData: BlogsDataType = BlogsDataJson;
+  const params = useParams();
+  const blogId = params.id as string;
+  const blogData = BlogsData.entries[blogId];
```

Comment:
add an optional check for the blogid if its not available

Reply:
There is a check on line 13 for the blog id, is that what you mean?

Reply:
thats okay